app/views/ProjectsView/components/AddEditProjectDialog/index.scss

```diff
@@ -3,4 +3,7 @@
     margin: 8px;
     padding: 0 16px 0 0;
   }
+}
+.edit-dialog__section-title {
```

Comment:
for future development, if `edit-dialog` gets more elements, we should split it via

```
.edit-dialog {
  &__section-title {
    ...
  }
  &__<some other element> {
    ...
  }
}
```